usecase/usecase-wrt-android-tests/tests.full.xml

```diff
@@ -69,6 +69,8 @@
       </testcase>
       <testcase component="Crosswalk Use Cases/WRT" execution_type="manual" id="LZMACompressionUI" platform="all" priority="P1" purpose="LZMACompressionUI Test" status="designed" type="functional_positive">
       </testcase>
+      <testcase component="Crosswalk Use Cases/WRT" execution_type="manual" id="SharedModeLibraryDownload" platform="all" priority="P0" purpose="SharedModeLibraryDownload Test" status="designed" type="functional_positive">
```

Comment:
Why status is design here? As feature status is resolved, please update to approve